### PR TITLE
Convert Crontab cheatsheet to JSON

### PR DESCRIPTION
**Pull request by AlfonzM**
This is my attempt at converting the Crontab cheat sheet into the new JSON format (in reference to #1155).

Screenshots:
screen shot 2015-06-14 at 11 55 02 pm
screen shot 2015-06-14 at 11 54 49 pm

### CODE REVIEW

**Comment by javathunderman** [comment]
cc @jagtalon

**Comment by moollaza** [comment]
/cc @abeyang 

This one looks OK, but the content is a bit of an edge-case 

Also, I just noticed that I need to add some CSS to maintain the width of the description in the expanded state :)

**Comment by moollaza** [comment]
Also, @AlfonzM thanks a lot for contributing, we really appreciate it! We'll be sure to review this shortly and give you some feedback!

**Comment by zachthompson** [comment]
When I looked at the old one initially it looked challenging to convert.  We might consider remove all /path/to/command entries to slim it down since we're really discussing date/time specifications.  Maybe a single entry under fields explaining that they all take a command to execute.

**Comment by moollaza** [comment]
@zachthompson that's a great suggestion. 

@AlfonzM let's remove the `path/to/command` from the examples.

File: share/goodie/cheat_sheets/crontab.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            "key" : "0-23",
+            "val" : "hour"
+        }, {
+            "key" : "1-31",
+            "val" : "day of month"
+        }, {
+            "key" : "[1-12] or [first 3 letters]",
+            "val" : "month"
+        }, {
+            "key" : "[0-7] or [first 3 letters (0 or 7 is Sun)]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/crontab.json]: Let's move the "(0 or 7 is Sun)" to the `value`

**Comment by mintsoft** [comment]
@AlfonzM Just checking in, are you still interested in finishing this off?

File: share/goodie/cheat_sheets/crontab.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+    "id": "crontab_cheat_sheet", 
+    "name": "Crontab Cheat Sheet",
+    "description": "Commands are executed by cron when the minute, hour, and month of year fields match the current time, and at least one of the two day fields (day of month, or day of week) match the current time. A field may be an asterisk (*), which will always match.",
+    "metadata": { 
+        "sourceName": "Crontab Cheat Sheet",
+        "sourceUrl": ""
+    },
```
Comment (moollaza) [inline share/goodie/cheat_sheets/crontab.json]: We've *just* added support for CheatSheet Templates that change the layout to favour certain kinds of data. Let's add `"template_type": "terminal"` to this IA. It should make the keys much easier to read!

**Comment by moollaza** [comment]
@AlfonzM Ping! I'm just checking in to see if you're still interested in finishing this one up? We'd love to see this go live!

There's absolutely no rush, just wanted to make sure you're not stuck on anything.

**Comment by zekiel** [comment]
this will use the current IA page here: https://duck.co/ia/view/crontab_cheat_sheet

**Comment by GuiltyDolphin** [comment]
@AlfonzM @moollaza @zachthompson Situation with this one? I'd like to see this (and any other non-JSON cheat sheets) converted so they stay up-to-date with any changes to the controller.

**Comment by mintsoft** [comment]
@GuiltyDolphin I'm marking this as abandoned and closing it; if you want to pick it up feel free

**Comment by mintsoft** [comment]
@AlfonzM I've marked this as abandoned, if you'd like to finish it up etc feel free to ping us and we can reopen it for you :)

**Comment by javathunderman** [comment]
@moollaza seems like #1155 has been closed - the Abandoned label can be removed.

**Comment by GuiltyDolphin** [comment]
@javathunderman Looks like it hasn't been moved over yet! I'll leave it on until the Goodie has been ported. (I think I had a branch somewhere...)

**Pull request closed without merging**